src/js/__tests__/component_test/Headline.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Renderer from 'react-test-renderer';
+import Headlines from 'Headlines';
+
+test('Component render the article template', () => {
+  const article = {
+    author: 'celestine',
+    title: 'My story',
+    urlToImage: 'test.jpg',
+    publishedAt: '24/12/24',
+    isAuth: true,
+    url: 'http://localhost',
+  };
+  const funtest = () => {
+    return 1;
+  };
+  const i = 1;
+  const source = 'BBC';
+  const isAuth= true;
+  const component = Renderer.create(
+    <Article
+      key={i} id={i} author={article.author} title={article.title}
+      urlToImage={article.urlToImage} description={article.description}
+      publishedAt={article.publishedAt} url={article.url} source={source}
+      isAuth={isAuth}
+    />,
+  );
+  const tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('Value of text of each control should be same with props property', () => {
```

Comment:
... **should be same with props property** can be made more descriptive. Note this in the previous test and anywhere else it's repeated.